### PR TITLE
Changes to StrongLoop branding

### PR DESCRIPTION
**Pull request by crandmck**
Per issue #237

### CODE REVIEW

**Comment by dougwilson** [comment]
Can you include a screenshot of the home page, pretty please :)?

**Comment by crandmck** [comment]
Here you go: http://screencast.com/t/Twp8uKxBH

**Comment by dougwilson** [comment]
Cool. Based on the screenshot, LGTM :shipit:

File: index.html [hunk 1]
```diff
@@ -24,7 +23,12 @@
 </div>
 
 <div id="sponsorship">
-  <a href="http://strongloop.com/developers/events/"><img src="/images/expressjs-strongloop-training.svg" alt=""></a>
+
+  <a href="http://loopback.io/"><h3>LoopBack</h3>
+
+  Develop model-driven apps with Express-based framework.
```
Comment (Fishrock123) [inline index.html]: hmmm.. wording sounds off.

Comment (dougwilson) [inline index.html]: At the very least, it's missing the article `an` in the sentence.

Comment (Fishrock123) [inline index.html]: Yeah. I was trying to think of a better way to word it but my brain is fried haha

**Comment by altsang** [comment]
:+1: LGTM too

File: index.html (hunk 1 above)
Comment (dougwilson) [inline index.html]: "Develop model-driven apps with **an** Express-based framework." at least makes the sentence sound a little less weird, lol

**Comment by crandmck** [comment]
Yeah...  I was trying to be telegraphic in the wording, but now that I look at it again, it just sounds odd.  I'll just add "an" as you suggest.

**Comment by dougwilson** [comment]
I'm not sure what's going on, but the styles look all messed up on the live site. Can anyone else confirm?

**Comment by crandmck** [comment]
Try a hard reload.  The CSS was changed.  It looks OK to me.

**Comment by dougwilson** [comment]
I have been and I even loaded it in new browsers. It's actually fixed now, so it seems like I just need to be patient, as GitHub seems to not refresh all the files at once.

**Comment by dougwilson** [comment]
Because some of the pages still were serving the old top-right thing at the same time the CSS was messed up :)

**Comment by crandmck** [comment]
Weird.  Glad it sorted itself out.

**Pull request merged**